src/github.com/couchbase/sync_gateway/base/sharded_sequence_clock.go

```diff
@@ -371,13 +371,7 @@ func (p *ShardedClockPartition) getSequenceForOffset(offset uint16, size uint8)
 func (p *ShardedClockPartition) getSequenceUint16(offset uint16) (seq uint64) {
 
 	return uint64(p.value[offset+1]) |
-		uint64(p.value[offset])<<8 |
-		uint64(0)<<16 |
-		uint64(0)<<24 |
-		uint64(0)<<32 |
-		uint64(0)<<40 |
-		uint64(0)<<48 |
-		uint64(0)<<56
+		uint64(p.value[offset])<<8
```

Comment:
Given that get/set sequence are very high frequency operation, taking the placeholders out was a performance optimization.

There's another critical performance fix in sharded_sequence_clock in this PR (the map allocation).  I agree it would be ideal as a separate PR, but left it in this one to save time.